.gitignore

```diff
@@ -70,36 +211,86 @@ typings/
 
 # dotenv environment variables file
 .env
-.env.test
 
-# parcel-bundler cache (https://parceljs.org/)
-.cache
+### SublimeText ###
+# cache files for sublime text
+*.tmlanguage.cache
+*.tmPreferences.cache
+*.stTheme.cache
 
-# next.js build output
-.next
+# workspace files are user-specific
+*.sublime-workspace
 
-# nuxt.js build output
-.nuxt
+# project files should be checked into the repository, unless a significant
+# proportion of contributors will probably not be using SublimeText
+# *.sublime-project
 
-# vuepress build output
-.vuepress/dist
+# sftp configuration file
+sftp-config.json
 
-# Serverless directories
-.serverless/
+# Package control specific files
+Package Control.last-run
+Package Control.ca-list
+Package Control.ca-bundle
+Package Control.system-ca-bundle
+Package Control.cache/
+Package Control.ca-certs/
+Package Control.merged-ca-bundle
+Package Control.user-ca-bundle
+oscrypto-ca-bundle.crt
+bh_unicode_properties.cache
 
-# FuseBox cache
-.fusebox/
+# Sublime-github package stores a github token in this file
+# https://packagecontrol.io/packages/sublime-github
+GitHub.sublime-settings
 
-# DynamoDB Local files
-.dynamodb/
+### Vim ###
+# swap
+.sw[a-p]
+.*.sw[a-p]
+# session
+Session.vim
+# temporary
+.netrwhist
+# auto-generated tag files
+tags
 
-# Secrets
-.secret
+### Windows ###
+# Windows thumbnail cache files
+Thumbs.db
+ehthumbs.db
+ehthumbs_vista.db
 
-# Yarn (https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored)
-.yarn/*
-!.yarn/releases
-!.yarn/plugins
-!.yarn/sdks
-!.yarn/versions
-.pnp.*
+# Folder config file
+Desktop.ini
+
+# Recycle Bin used on file shares
+$RECYCLE.BIN/
+
+
+# Windows Installer files
+*.cab
+*.msi
+*.msm
+*.msp
+
+# Windows shortcuts
+*.lnk
+
+# No snapshot schemas
+src/migrations/schemas/snapshot-*
+
+# Docs
+/docs/
```

Comment:
Do we want to gitignore `docs` directories?

Reply:
will rm. don't know where this gitignore came from originally, but i pasted everything over from penumbra